lib/screens/tree_list/tree_list_view_model.dart

```diff
@@ -25,16 +29,12 @@ abstract class TreeListViewModelBase with Store, ViewModel {
 
   @observable
   bool isLoadingTrees = false;
-  bool hasInterNetConnection = false;
+  bool hasInternetConnection = false;
   List<Tree> newTrees = [];
 
   @override
   void init() {
-    connectionStatus.initialize();
-    //Listen for connection change
-    connectionStatus.connectionChange
-        .asBroadcastStream()
-        .listen(connectionChanged);
+    activateConnectionManager();
```

Comment:
Should already be initialized in the main file. 
Just import the manager with dependency injection